packages/main/src/Slider.js

```diff
@@ -158,6 +158,10 @@ class Slider extends SliderBase {
 		if (this._getInitialValue("value") === null) {
 			this._setInitialValue("value", this.value);
 		}
+
+		if (this.showTooltip) {
+			this._tooltipVisibility = "visible";
```

Comment:
could you please extract these as constants. In case we have more complex logic we would have these strings on several places

Reply:
Done